assets/js/blocks/product-gallery/index.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import icon from './icon';
+
+// @ts-expect-error: `registerBlockType` is a function that is typed in WordPress core.
+registerBlockType( metadata, {
```

Comment:
Good point @tjcafferkey ! By doing what you suggested, is it necessary mark the block as "experimental" in [`webpack-entries.js` file](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/bin/webpack-entries.js#L67)? Or it wouldn't be necessary?

Reply:
I had forgotten about this property in the webpack entries file, thanks for reminding me. I'd probably do it in both places similar to how the Product Collections block has been done at least then we're covered twice over.

It might also be worth just leaving a comment in this file that there is an additional check in the webpack entries file too so when it comes to being graduated we remember to remove both checks.